rholang/src/main/scala/coop/rchain/rholang/interpreter/dispatch.scala

```diff
@@ -53,7 +53,7 @@ class RholangAndScalaDispatcher[M[_]] private (
 
 object RholangAndScalaDispatcher {
 
-  def create[M[_], F[_]](
+  def createWithEmptyCost[M[_], F[_]](
```

Comment:
Why not write a more general `createWithCost(cost: Cost, ...)`. That would cover this and other cases.